### PR TITLE
Inventory Items API and Bug Fix on respond.map.

### LINKED ISSUE #8: Inventory Items API

### LINKED ISSUE #10: If one object is returned, respond.map will fail.
If the response has exactly 1 element, the:
```Ruby
      respond.map(&:to_h)
```
will fail with:
```
     NoMethodError:
       undefined method `map' for #<Blanket::Response:0x007f9d64938060>
```

### PR DESCRIPTION
**Pull request by michaelachrisco**
Fixed #8
Fixed #10

### CODE REVIEW

File: lib/gman_client.rb [hunk 1]
```diff
@@ -14,18 +14,36 @@ class << self
 
     def drivers
       respond = request.api.v1.drivers.get
```
Comment (josephbridgwaterrowe) [inline lib/gman_client.rb]: Do you agree that response would be a better name than respond? :smile:

File: spec/lib/inventory_item_response_spec.rb [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+      VCR.use_cassette('inventory_items') do
+        GmanClient.inventory_items
+      end
+    end
+
+    subject(:client_response) { response }
+    let(:driver_commission) { client_response.first }
+
+    it 'client response will not be empty' do
+      expect(client_response.empty?).to be_falsey
```
Comment (josephbridgwaterrowe) [inline spec/lib/inventory_item_response_spec.rb]: Couldn't this be:

`is_expected`

File: lib/gman_client.rb (hunk 1 above)
Comment (michaelachrisco) [inline lib/gman_client.rb]: How about reply? 
or hit/attack if we want to get violent.

File: spec/lib/inventory_item_response_spec.rb (hunk 2 above)
Comment (michaelachrisco) [inline spec/lib/inventory_item_response_spec.rb]: I guess I could do: 
```
    it 'client response will be truthy' do
      is_expected.to be_falsey
    end
```
But that will bring us back to our original problem of the test always passing. what do you have in mind?

Comment (michaelachrisco) [inline spec/lib/inventory_item_response_spec.rb]: Oh I see! Here is something I could do: 
```it { is_expected.not_to be_empty }```

Comment (michaelachrisco) [inline spec/lib/inventory_item_response_spec.rb]: Fixed with 7bc1db4
Also added to other applicable tests.

File: lib/gman_client.rb (hunk 1 above)
Comment (michaelachrisco) [inline lib/gman_client.rb]: fixed with 1d9b6c1

File: spec/lib/driver_response_spec.rb [hunk 3]
```diff
@@ -16,7 +16,7 @@
     let(:driver) { client_response.first }
 
     it 'client response will not be empty' do
-      expect(client_response.empty?).to be_falsey
+      is_expected.not_to be_empty
```
Comment (josephbridgwaterrowe) [inline spec/lib/driver_response_spec.rb]: Can you remove the client_response from the subject now?

Comment (josephbridgwaterrowe) [inline spec/lib/driver_response_spec.rb]: Please ignore the previous comment, I see that it is referenced in a nested describe.

Comment (michaelachrisco) [inline spec/lib/driver_response_spec.rb]: :+1: 1d9b6c1

**Pull request merged**